Public/Src/Cache/ContentStore/Library/Stores/FileSystemContentStoreInternal.cs

```diff
@@ -2275,7 +2266,7 @@ public async Task<EvictResult> EvictAsync(Context context, ContentHashWithLastAc
 
                     if (code != PlaceFileResult.ResultCode.Unknown)
                     {
-                        return new PlaceFileResult(code, contentSize);
+                        return new PlaceFileResult(code, contentSize).WithLockAcquisitionDuration(contentHashHandle);

```

Comment:
>.W [](start = 69, length = 2)

nit: Slight style-improvement to put this indented on the next line to signify an operation is happening on the result.